app/src/main/java/de/danoeh/antennapod/fragment/preferences/NetworkPreferencesFragment.java

```diff
@@ -174,4 +223,162 @@ private void showUpdateIntervalTimePreferencesDialog() {
         }
         return entries;
     }
+
+
+    public class ViewDialog {
```

Comment:
Please move this to a new file. We have some similar dialogs in `AntennaPod/app/src/main/java/de/danoeh/antennapod/dialog/`. Also, please rename to something more meaningful (maybe something like `FeedRefreshIntervalDialog`?)